### DISCUSSION TITLE
Check overflow on <Text/> with truncate or lineClamp

### DISCUSSION THREAD

**Discussion by zangerls** [mantinedev/mantine#4175] [Q&A] [answered]
Hi.

I cannot seem to figure out a way to check whether or not a `Text` component with either `truncate` or `lineclamp` is overflowing or not.

The goal would be to enable/disable a `Tooltip` on the overflowing `Text` component with its contained text.

Thanks!

**Comment by yverby** [comment, marked as answer]
You can compare `element.scrollHeight > element.clientHeight` if the value is `true` then your element is clipped
